pkg/shell/shell.less

```diff
@@ -37,6 +37,7 @@ html.index-page body {
 
 .curtains-ct {
     top: auto;
+    position: inherit;
```

Comment:
This makes the position dependent on the parent element. We don't want that. (If the parent changes position, it would break the positioning of the curtain.)

It should be `position: static;` instead, which sets the position directly.

Reply:
`static` seems to be exactly the same as `inherit`.  `relative` also gives the same result.